runtime/src/main/java/org/corfudb/runtime/MultiCheckpointWriter.java

```diff
@@ -105,8 +105,8 @@ public long appendCheckpoints(CorfuRuntime rt, String author,
                 }
                 final long mapCpEnd = System.currentTimeMillis();
 
-                log.info("appendCheckpoints: took {} ms to checkpoint map {}",
-                        mapCpEnd - mapCpStart, streamId);
+                log.info("appendCheckpoints: took {} ms to checkpoint {} entries for map {}",
+                        mapCpEnd - mapCpStart, ((SMRMap) map).size(), streamId);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "SMRMap"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)